packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -528,6 +531,14 @@ contract Bridge is EssentialContract, IBridge {
 
         _storeContext({ msgHash: msgHash, from: message.from, srcChainId: message.srcChainId });
 
+        if (
+            message.data.length >= 4 // msg can be empty
+                && bytes4(message.data) != IMessageInvocable.onMessageInvocation.selector
+                && message.to.isContract()
```

Comment:
Very niche case, but you could think of some attack where `message.to` is not yet a contract, and then somebody does a bridge transaction and in the mean time `message.to` would be a contract and then any funds of the bridge transaction would be locked because this reverts. I think safer to just let this return false instead of throwing so it can be recalled.